iam.tf

```diff
@@ -31,22 +32,25 @@ resource "aws_iam_role_policy" "codebuild_policy" {
           "logs:CreateLogStream",
           "logs:PutLogEvents"
         ],
-        Resource = "*"
+        Resource = "arn:aws:logs:${var.region}:${data.aws_caller_identity.current.account_id}:log-group:/aws/codebuild/${var.environment}-${var.pipeline_name}-codebuild*"
       },
       {
         Effect   = "Allow",
         Action   = [
           "s3:GetObject",
           "s3:PutObject"
         ],
-        Resource = "*"
+        Resource = [
+          "${aws_s3_bucket.codebuild_artifacts.arn}",
```

Comment:
For Get & Put objects, you can just leave the `"${aws_s3_bucket.codebuild_artifacts.arn}/*"` line

Reply:
Got it. I will include it in my next revision.